CircuitPython_Essentials/CircuitPython_I2C_Scan.py

```diff
@@ -1,15 +1,24 @@
 # CircuitPython demo - I2C scan
+# 
+# If you run this and it seems to hang, try manually unlocking
+# your I2C bus from the REPL with
+# ` >>> import board `
+# ` >>> board.I2C().unlock() `
 
 import time
-
 import board
 
 i2c = board.I2C()
 
 while not i2c.try_lock():
     pass
 
-while True:
-    print("I2C addresses found:", [hex(device_address)
-                                   for device_address in i2c.scan()])
-    time.sleep(2)
+try:
+
+    while True:
+        print("I2C addresses found:", [hex(device_address)
+            for device_address in i2c.scan()])
```

Comment:
This line needs two more spaces in it to line up with the rest of the print statement.

The other option is that the print statement fits on one line. If you'd rather, you can put this on the same line as `print("I2C addresses found:"....`.

Reply:
Leaving it on two lines, added the spaces, that's the way it was originally.